shared/domain/src/main/java/gov/va/vro/model/mas/MasAnnotation.java

```diff
@@ -0,0 +1,53 @@
+package gov.va.vro.model.mas;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+/** @author warren @Date 10/5/22 */
+@NoArgsConstructor
+@Getter
+@Setter
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class MasAnnotation {
+  @JsonProperty("annot_type")
```

Comment:
Not sure why we are repeating these. @JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class) does not work?